### ISSUE TITLE
Rapidly switch between playlists cause audio to glitch for a few seconds

### ISSUE THREAD

**Issue by A6sT** [Quaver/Quaver#4266]
### Describe the bug

Switching between playlist causes the game to repeat the audio of the displayed preview map of the playlist. Note that this only happening on playlist.

Here's a video showing the issue (I started by switching using only 1 key between playlist, then pressing 2 keys at the same time)
https://github.com/user-attachments/assets/76978600-ab68-4412-ba38-093d63e2abcc

This can be reproduced on any versions (canary and stable have been tested before reporting this issue)

### To Reproduce

1. Go to the playlist section
2. Switch between two playlist using at least two of the arrow keys

### Expected behavior

The audio should only start playing once for the selected playlist after rapidly switching between them

### Log Files

it's an audio issue so there's not really any need for logs

**Comment by Warp9000** [comment]
I cant replicate. Does this only happen with imported etterna lists?

**Comment by A6sT** [comment]
Not really. Just to be sure, I created a playlist containing only one ranked song, and I managed to reproduce the issue

**Comment by A6sT** [comment]
can't reproduce consistantly, so i'll close
